test/Microsoft.AspNet.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -2,13 +2,18 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 #if DNX451
```

Comment:
This is looking good. Thanks for adding the HttpsConnectionFilterOptions.

Recently we enabled these HttpsConnectionFilterTests on CoreClr since SslStream is now available for that.

Can you rebase this PR on dev and verify that client certificates work with CoreClr? Thanks!